src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+      <div class="stars_star"></div>
+    </div>
```

Comment:
use double underscore [Typical BEM Mistakes](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-a-modifier-instead-of-an-element)